angular.json

```diff
@@ -23,18 +23,22 @@
             "polyfills": "src/polyfills.ts",
```

Comment:
Were these changes a result of your linters? What linter were you using?

Reply:
I am using Prettier but I think that most of them could cause I added a section in the options of the build